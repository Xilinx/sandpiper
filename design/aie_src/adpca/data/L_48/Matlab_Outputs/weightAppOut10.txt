5.448272705e+02 -8.201828003e+02
2.533312082e+00 4.822645187e+00
-1.303812027e+00 1.521287203e+00
