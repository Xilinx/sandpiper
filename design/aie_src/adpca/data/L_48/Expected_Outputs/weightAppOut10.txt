5.448120728e+02 -8.201486206e+02 
2.533123016e+00 4.822591782e+00 
-1.303741455e+00 1.521211624e+00 
